packages/shared/src/components/sidebar/Sidebar.tsx

```diff
@@ -177,6 +193,14 @@ export default function Sidebar({
     toggleSidebarExpanded();
   };
 
+  const trackAndShowSubmitArticle = () => {
+    trackEvent({
```

Comment:
Is it possible to add here a flag that indicates whether the user has access to the feature or not?
If not we can find a more appropriate place for it.
I'd like to separate between conversion rates of people who have access to those who don't

Reply:
If added an extra: { has_access: boolean } is that ok?

Reply:
Yes. In the extra field